### PR TITLE
Add created-by label to CPRs

### PR DESCRIPTION
**Pull request by jasonwzm**

### CODE REVIEW

File: asmcli/commands/help.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ help_subcommand() {
 }
 
 version_message() {
-  local VER; VER="${MAJOR}.${MINOR}.${POINT}-asm.${REV}+config${CONFIG_VER}";
```
Comment (zerobfd) [inline asmcli/commands/help.sh:39]: Changing it here is going to change it everywhere which will break a ton of things. Set a variable to $(version_message) before the kpt setter and then do `var="asmcli-${mess_var/+/.}"`` or something similar

**Review by zerobfd — CHANGES_REQUESTED** [review]

File: asmcli/commands/help.sh (hunk 1 above)
Comment (jasonwzm) [reply asmcli/commands/help.sh]: Done.

**Review by zerobfd — APPROVED** [review]
If you tested it locally and it works then LGTM

**Comment by jasonwzm** [comment]
Manually tested. Label is added with expected value.

**Pull request merged**